public/assets/styles/routes/datasets.scss

```diff
@@ -94,9 +94,24 @@
     }
   }
 
+  .dataset-details-content {
+    display: flex;
+
+    & > .details-content-column {
+      flex: 1 0 75%;
+
+      &:last-of-type {
+        flex: 1 0 23%;
+        top: -30px;
+
```

Comment:
Are these bonus line breaks in the CSS intentional? If so what style logic are they following?